File: freshop/Models/CartItems.cs
using System;
namespace freshop.Models
{
    public class CartItems
    {
        public int id { get; set; }
        public string cart_guid { get; set; }
        public int product_id { get; set; }
        public string product_guid { get; set; }
        public int quantity { get; set; }
        public int items { get; set; }
    }
}

File: freshop/wwwroot/assets/cartitems.js
const cartItems = document.getElementById('cart_items');

if (localStorage.getItem("freshop-guid")){
    const userGuid = localStorage.getItem("freshop-guid");
    fetch('http://localhost:63492/api/CartItems/'+userGuid)
    .then(response => response.json())
    .then(json => {
        cartItems.textContent = json.items;
        cartItems.parentElement.href = 'cart.html?guid='+json.cart_guid;
    });
}